runner/runner.go

```diff
@@ -1343,17 +1343,22 @@ retry:
 		}
 		builder.WriteRune(']')
 	}
-
 	var faviconMMH3 string
 	if scanopts.Favicon {
-		faviconMMH3 = fmt.Sprintf("%d", stringz.FaviconHash(resp.Data))
-		builder.WriteString(" [")
-		if !scanopts.OutputWithNoColor {
-			builder.WriteString(aurora.Magenta(faviconMMH3).String())
+		favURL := fmt.Sprintf("%s://%s/favicon.ico", protocol, URL.Host)
+		favreq, _ := hp.NewRequest(method, favURL)
```

Comment:
I think we can reuse the original request (it's not referenced anymore in the code) and avoid memory allocations:
```go
req.URL.Path = "/favicon.ico"
if faviconResp, favErr := hp.Do(req, httpx.UnsafeOptions{}); favErr == nil {
...
```

Reply:
Done